src/components/Feed.tsx

```diff
@@ -1,13 +1,56 @@
-import React from "react";
+import React, { useEffect } from "react";
+import * as sdk from "../services/sdk";
 import NewPost from "./NewPost";
 import PostList from "./PostList";
+import { useAppDispatch, useAppSelector } from "../redux/hooks";
+import * as server from "../services/server";
+import {
+  addNewFeedItems,
+  addNewFeedtoMainFeed,
+} from "../redux/slices/feedSlice";
+import { BaseFilters } from "@unfinishedlabs/sdk/dist/types/social/search";
+import { Button } from "antd";
 
 const Feed: React.FC = () => {
+  const [subscriptionID, setSubscriptionID] = React.useState<string>("");
+
+  const dispatch = useAppDispatch();
+  const newFeed = useAppSelector((state) => state.feed.newFeed);
+
+  useEffect(() => {
+    subscribe();
+    dispatch(addNewFeedtoMainFeed());
```

Comment:
there is more than one feed? Like a personal feed or custom and public feed?

Reply:
I've made a distinction between feed data that comes in that is new and the main feed of already loaded data. Doing so makes it easy to allow the current loaded feed data to not be constantly pushed down by new incoming data. Instead that new feed data is loaded in when the page is refreshed or when the "load new content" button is pressed.

However beyond this single distinction there is no separating of feed data for personal or public. All of that would/will be done by filtering the existing feed.